samples/ClientSample/HubSample.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.IO.Pipelines;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.AspNetCore.Sockets.Client;
+using Microsoft.Extensions.Logging;
+
+namespace ClientSample
+{
+    internal class HubSample
+    {
+        public static async Task MainAsync(string[] args)
+        {
+            var baseUrl = "http://localhost:5000/hubs";
+            if (args.Length > 0)
+            {
+                baseUrl = args[0];
+            }
+
+            var loggerFactory = new LoggerFactory();
```

Comment:
No argument here. Wanted to start with everything I thought we'd need though so it's easier to clean-up.